2-mandatory/3-financial-times.js

```diff
@@ -14,6 +22,8 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
+    const sorted = allArticleTitles.sort((a, b) => a.length - b.length);
```

Comment:
One small problem here - this code will give us the article with the fewest characters, but we want the article with the fewest words. It's a good idea though! :)